types/rdfjs__data-model/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for @rdfjs/data-model 2.0
+// Project: https://github.com/rdfjs-base/data-model
+// Definitions by: Jesse Wright <https://github.com/jeswr>
+//                 Ruben Taelman <https://github.com/rubensworks>
+//                 Andy Davison  <https://github.com/andydavison>
```

Comment:
If you won't mind. I contributed a significant portion of RDF/JS-related types on DT

```suggestion
//                 Andy Davison  <https://github.com/andydavison>
//                 tpluscode <https://github.com/tpluscode>
```